tt_eager/tt_dnn/op_library/conv/kernels/conv_bmm_tilize_col_major_out_blocks.cpp

```diff
@@ -195,7 +205,20 @@ void MAIN {
                     }
                     #endif
                     unpack_reconfig_data_format_srca(in1_cb_id, in0_cb_id);
-                    tilize_in(in0_cb_id, in0_subblock_h, in0_block_w, in0_num_subblocks, tilized_in0_cb_id);
+
+                    // Configure to RowMajor for tilize (similar to add bcast for bias)
+                    MATH(( llk_math_pack_sync_init<SYNC>()  ));
+                    PACK(( llk_pack_dest_init<SYNC, DstTileFaceLayout::RowMajor, false>()  ));
+
+                    tilize_in(in0_cb_id, in0_subblock_h, in0_block_w, in0_num_subblocks_read, tilized_in0_cb_id);
```

Comment:
why would we need 2 tilize calls? 

reader 0 fills bottom half of each input block
reader 1 fills  top-half of each input block

at most we'd need 2 wait_for_tiles in compute since both readers need to signal they filed up their block, which could be via two CB (although they would map to the same address space)